src/content/caseStudies/contenda-share-pear/index.mdx

```diff
@@ -0,0 +1,70 @@
+
+Grady Salzman, software engineer, says, “Depending on your team size, Contenda could enable others to make content for you. You could have someone who is spending all their time and effort on recording videos. Someone else could take those videos and write a blog post in their own thoughts, tone, and voice. If you’re a smaller team, it could enable you to release faster.”
+
+However, for all the ways Contenda was theoretically supposed to make collaboration and content creation easier, it needed to cross one practical hurdle: being able to share blog drafts publicly with other people. Contenda solved this by choosing Astro and Netlify as their solution.
+
+</div>
+
+## Challenges
+
+Contenda is a team of less than 10 people. They often need to share blogs with each other in order to address pain points and conduct reviews but they couldn’t do these things without sharing their emails and API keys with one another. This also posed a security problem for customers that they knew they had to solve eventually. What would eventually be known and open-sourced as Share Pear was created to resolve both issues for internal and external users.
+
+## Requirements
+
+Share Pear needed to be built without requiring backend changes since Contenda did not want to overload themselves by managing additional complexity. Ideally, it also needed to be cost-effective but preferably free. Fortunately for the team, they were able to assess and select Astro as their solution all in a day’s time thanks to quick work from Cassidy Williams.
+
+Cassidy is Contenda’s CTO. Having previously worked at Netlify plus being an active developer in the Astro community, she saw an opportunity to use Astro with Netlify’s on-demand builders. In addition to meeting their requirements, the main benefit of choosing Astro was its flexibility. With Astro, you can choose to have React or HTML and CSS. The latter enables non-developer team members, such as a product designer, to make changes without requiring developer intervention.
+
+Contenda also knew they had to use Netlify’s on-demand builders. Astro was the obvious choice because it had a component that both natively supported Netlify’s builders and rendering markdown to HTML. It also helped that Contenda had previously chosen Astro for their main marketing website so the technology was familiar enough for the rest of the team to jump in.
+
+## Solution and Implementation
+
+At first, it was hard for the team to get started because there were barely any written tutorials about using Astro together with Netlify’s on-demand builders let alone just the latter. Their team ended up going into Astro’s Discord to enlist the help of the core team and the community. From there, building was fast for the team at Contenda because the developer experience was good.
```

Comment:
```suggestion
Contenda found success in Astro’s Discord by enlisting the help of the core team and the community. From there, building was fast because the developer experience was good.
```